bigtable/use-cases/fraudDetection/README.md

```diff
@@ -84,4 +84,54 @@ The current Cloud Bigtable instance configuration does not provide any replicati
+|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cloud Bigtable Instance          | featurestore-{RANDOM\_ID}                                                                                                                                                                                     |
+| Cloud Bigtable Table             | customer-information-{RANDOM\_ID}                                                                                                                                                                             |
+| Cloud Bigtable Column Family     | demographics, history                                                                                                                                                                                         |
+| Cloud Pubsub Input Topic         | transaction-stream-{RANDOM\_ID}                                                                                                                                                                               |
+| Cloud Pubsub Output Topic        | fraud-result-stream-{RANDOM\_ID}                                                                                                                                                                              |
+| Cloud Pubsub Output Subscription | fraud-result-stream-subscription-{RANDOM\_ID}                                                                                                                                                                 |
+| Google Storage Bucket            | fraud-detection-{RANDOM\_ID}                                                                                                                                                                                  |
+| Google Storage Objects           | 1. temp/ directory (for temporary dataflow generated files) <br/> 2. testing_dataset/ directory <br/>3. training_dataset/ directory <br/> 4. ml_model/ directory                                              |
+| VertexAI Model                   | fraud-ml-model-{RANDOM\_ID}                                                                                                                                                                                   |
+| VertexAI Endpoint                | **Name:** fraud-ml-model-ep-{RANDOM\_ID} <br/> **Id:** Determined in runtime, stored in Scripts/ENDPOINT\_ID.output                                                                                           |
+| Dataflow Load CBT                | **Name:** load-customer-demographics-{RANDOM\_ID}, load-customer-historical-transactions-{RANDOM\_ID} <br/> **Description:** Creates batch jobs that load both historical and demographic data from GS to CBT |
```

Comment:
Can we write `CBT` out as `Cloud Bigtable`? Also is `Dataflow Load CBT` a job?

Are these two different resources? If so can we put them as follows inside the cell:

load-customer-demographics-{RANDOM\_ID}
_(batch job that loads demographic data from GS to CBT)_

load-customer-historical-transactions-{RANDOM\_ID} 
_(batch job that loads historical data from GS to CBT)_